File: educational-core/src/com/jetbrains/edu/coursecreator/CCAnswerPlaceholderActionGroup.java
package com.jetbrains.edu.coursecreator;

import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.actionSystem.DefaultActionGroup;
import com.intellij.openapi.project.DumbAware;
import org.jetbrains.annotations.NotNull;

@SuppressWarnings("ComponentNotRegistered") // educational-core.xml
public class CCAnswerPlaceholderActionGroup extends DefaultActionGroup implements DumbAware {
  @Override
  public void update(@NotNull AnActionEvent e) {
    CCUtils.updateActionGroup(e);
  }
}

File: intellij-plugin/educational-core/src/com/jetbrains/edu/coursecreator/CCAnswerPlaceholderActionGroup.kt
package com.jetbrains.edu.coursecreator

import com.intellij.openapi.actionSystem.ActionUpdateThread
import com.intellij.openapi.actionSystem.AnActionEvent
import com.intellij.openapi.actionSystem.DefaultActionGroup
import com.intellij.openapi.project.DumbAware
import com.jetbrains.edu.coursecreator.CCUtils.updateActionGroup

class CCAnswerPlaceholderActionGroup : DefaultActionGroup(), DumbAware {

  override fun getActionUpdateThread(): ActionUpdateThread = ActionUpdateThread.EDT

  override fun update(e: AnActionEvent) {
    updateActionGroup(e)
  }
}
